### ISSUE TITLE
Request for data

### ISSUE THREAD

**Issue by serendipity800** [VincentHancoder/AToM#2]

**Comment by Maddog241** [comment]
Hi, @serendipity800 

I have located the corresponding datasets based on the logs and uploaded them to Google Drive [link](https://drive.google.com/drive/folders/1WYu9X7EYhyahm2uDfo7mE6AZMsUQ6P0D). They correspond to the three subtasks: temporal.zip, integrity.zip, and frequency.zip, including the corresponding prompts, motion tokens, and stick figure renderings for observing motion quality.

About the dataset scale mentioned in our paper: The 80k figure represents a theoretical combinatorial count (pairwise combinations of 10 candidate outputs per prompt). The data I have shared is a filtered set of 3 pairs per prompt. And we used these exact data for finetuning the 3 subtasks in our paper. 

From my current perspective, the quality of this data is not good. These motions were generated by our basemodel MotionGPT, which has limited zero-shot capability, so the quality of the generated motions given new prompts is suboptimal (you can download one of the subset and see the stick figure rendering). Additionally, the annotations from GPT-4o are not very accurate (the API's performance is not stable).  If you use it to fine-tune a stronger model, the base model may be misled by MotionGPT's suboptimal distribution, resulting in decreased quality. Therefore, I suggest that if you intend to follow this multimodal large model fine-tuning approach, you could use these prompts to generate motions with your own base model, render them into SMPL format, and then score them using a strong open-source multimodal model (such as Qwen-VL) to label smpl renderings and create preference pairs. Best regards.
